src/shared/managers/SubscriptionManager.ts

```diff
@@ -834,7 +833,6 @@ export class SubscriptionManager {
      */
 
     const isPushEnabled = !!(
```

Comment:
Removing the playerId check does change the definition of this, I won't say `PushSubscriptionState.subscribed ` is an accurate name for this then. `Eligible` would be a better wording, if we really don't want to change this playerId check to a subscriptionId check.

We should also check how this was used to see if this change will cause any side effects.